engine/src/main/java/org/terasology/config/flexible/Setting.java

```diff
@@ -98,7 +99,12 @@ default void resetValue() {
     boolean hasSubscribers();
 
     /**
-     * Parses the String based on the type of this {@link Setting}, and sets it.
+     * Sets the value of this {@link Setting} from the value's JSON representation.
```

Comment:
`* Sets the value of this {@link Setting} from a JSON representation as a string.`